lib/routes/account.js

```diff
@@ -217,6 +217,7 @@ module.exports = function (
             .then(
               function (result) {
                 sessionToken = result
+                metricsContext.save(sessionToken, form.metricsContext, config)
```

Comment:
I wonder if `metricsContext.save` could take any arbitrary token type, rather than just a sessionToken.  Then we could use the same mechanism for storing info about the keyFetchToken as well.